app/templates/suppliers/edit_supplier_organisation_size.html

```diff
@@ -0,0 +1,61 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Organisation size – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {% with items = [
+    {
+      "link": "/",
+      "label": "Digital Marketplace"
+    },
+    {
+       "link": url_for('.dashboard'),
+       "label": "Your account"
+    },
+    {
+      "link": url_for('.supplier_details'),
+      "label": "Company details"
```

Comment:
Did this breadcrumb text come from the prototype?  In the "registered address" branch we were going with "Supplier details" for the breadcrumb text, as that's the heading on the page (I think). 

But that is going to change with an upcoming ticket.

Either way we should agree what the breadcrumb text is and be consistent.

Reply:
`Company details` is indeed from the prototype.

Reply:
@constancecerf any input? :)